Azure-Samples/Cognitive-Services-Voice-Assistant
# Add Change Log to UWPVoiceAssistantSample

## Purpose
* Add a Change Log to the existing UI
* Change Log is within a Scrollviewer which scrolls to the most recent item inserted in the bottom
* Scrollviewer contains multiple textblocks enabling future color coding implementation for various log levels.

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

## How to Test
*  Open solution
* Enter valid configuration settings in config.json
* Push to Talk or Keyword Activate.
* Application log messages should appear in the Change Log Window
* Change Log Window should refocus to the bottom where most recent textblock was created
